app/styles/_notifications.less

```diff
@@ -0,0 +1,66 @@
+// fixes positioning that doesn't quite work w/core patternfly styles
+.navbar-pf-alt .nav .nav-item-iconic .badge {
+  font-size: 9px;
+  margin: -15px 0 0 -12px;
+  min-width: 10px;
+  min-height: 10px;
+}
+
+notification-drawer-wrapper {
+  .drawer-pf {
+    height: calc(~"100vh - 80px");//to create a 20px offset bottom
+    top: @navbar-os-header-height-desktop - 10;
+    z-index: @zindex-navbar-fixed + 1; // was 1050
+    .tech-preview & {
+      top: @navbar-os-header-height-desktop + @tech-preview-banner-height - 10;
+    }
+  }
+  // the whole block is clickable, but need to set pointer on all of these :/
+  .drawer-pf-notification,
+  .drawer-pf-notification-info,
+  .drawer-pf-notification-message {
+    cursor: pointer;
+  }
+  .drawer-pf-notification {
+    padding: 0px;
+  }
+  .drawer-pf-notification.ng-leave {
+    //transition:0.25s linear all;
+    //opacity:1;
```

Comment:
Enabling these again, I think the slight animation out helps when events are *almost* the same.